Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builder image
-FROM golang:1-alpine as builder
+FROM golang:1.14-alpine as builder
```

Comment:
Use latest, please.

Reply:
Then let's leave it as is because `golang:1-alpine` is the latest.